internal/website/content/howto/server/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Server"
+date: 2019-06-21T10:36:43-07:00
+draft: true
+showInSidenav: true
+---
+
+The Go CDK's `server` package provides a pre-configured HTTP server with diagnostic hooks for request logging, health checks, and trace exporting via OpenCensus. These guides will show you how to start up and shut down the server, as well as how to work with the request logging, health checks, and trace exporting.
```

Comment:
nit: wrap to 80 columns here and elsewhere

Reply:
Strongly suggest using `mdformat`:

http://go/mdformat

There's no external alternative that we could find, but it's pretty handy.